include/onnxruntime/core/graph/graph.h

```diff
@@ -283,8 +283,12 @@ class Node {
   void ToProto(ONNX_NAMESPACE::NodeProto& proto) const;
 
   /** Call the provided function for all explicit inputs, implicit inputs, and outputs of this Node.
-      If the NodeArg is an explicit or implicit input, is_input will be true when func is called. */
-  void ForEachDef(std::function<void(const onnxruntime::NodeArg&, bool is_input)> func) const;
+      If the NodeArg is an explicit or implicit input, is_input will be true when func is called. 
+	  @param include_missing_optional_defs Include NodeArgs that are optional and were not provided 
```

Comment:
>	 [](start = 0, length = 1)

nit: tabs. Please use .clangformat

Reply:
Weird. I do use clangformat on all saves so not sure where the tabs came from.

---
In reply to: [252390544](https://github.com/Microsoft/onnxruntime/pull/410#discussion_r252390544) [](ancestors = 252390544)